Automattic/wp-calypso
# Hosting Metrics: First pass at 'Cache hit/miss ratio' and 'PHP vs. static' pie charts

See https://github.com/Automattic/dotcom-forge/issues/3276

## Proposed Changes

First pass at adding 'Cache hit/miss ratio' and 'PHP vs. static' pie charts:

<img width="1279" alt="image" src="https://github.com/Automattic/wp-calypso/assets/36432/3170c8b7-5f3a-4fe9-b8e9-72a7d555ee96">


However, the first isn't displaying in the picture because of p1691106078444819-slack-C7YPW6K40

## Testing Instructions

1. Navigate to `/site-monitoring/<site>`
2. See how the pie charts look.
